### PR TITLE
[TR] PIM-4874: add the possibility to unselect any row of a datagrid

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                  | A
| ------------------ | ---
| Migration script?  | -
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | -

### CODE REVIEW

**Comment by grena** [comment]
We need it on 1.4? Anyway, GTM

**Comment by solivier** [comment]
GTM

File: features/Context/Page/Base/Grid.php [hunk 1]
```diff
@@ -778,7 +779,11 @@ public function selectRow($value)
                     );
                 }
 
-                $checkbox->check();
+                if (true === $check) {
```
Comment (phaseinducer) [inline features/Context/Page/Base/Grid.php]: Do we use identity comparators for method parameters too ? Seems weird to me...

**Comment by jjanvier** [comment]
@grena for https://github.com/akeneo/pim-enterprise-dev/pull/1256

**Pull request merged**